nixos/modules/services/cluster/kubernetes/default.nix

```diff
@@ -222,14 +245,9 @@ in {
     })
 
     (mkIf cfg.kubelet.enable {
-      virtualisation.docker = {
-        enable = mkDefault true;
-
-        # kubernetes needs access to logs
-        logDriver = mkDefault "json-file";
-
-        # iptables must be disabled for kubernetes
-        extraOptions = "--iptables=false --ip-masq=false";
+      virtualisation.containerd = {
+        enable = true;
```

Comment:
Do we want to keep a `mkDefault` for this so it can be disabled?

Reply:
done (03ae1817121261f31e351aea2a4446827f967715)